src/cleanfigure.m

```diff
@@ -6,8 +6,8 @@ function cleanfigure(varargin)
 %   CLEANFIGURE('handle',HANDLE,...) explicitly specifies the
 %   handle of the figure that is to be stored. (default: gcf)
 %
-%   CLEANFIGURE('minimumPointsDistance',DOUBLE,...) explicitly specified the
-%   minimum distance between two points. (default: 1.0e-10)
+%   CLEANFIGURE('targetResolution','WxH@Res',...) explicitly specify the
+%   target resolution of the path simplification. (default: '15x9@600')
```

Comment:
I think we need a bit more explanation on this parameter to make sense of the three numbers that are given. I think it also should be mentioned that the dimensions of the 2 first ones are "length" (e.g. centimeter or inch) and the last is "points per [same unit]".